cluster_tools/__init__.py

```diff
@@ -24,12 +25,26 @@ class WrappedProcessPoolExecutor(ProcessPoolExecutor):
     def __init__(self, **kwargs):
         new_kwargs = get_existent_kwargs_subset(PROCESS_POOL_KWARGS_WHITELIST, kwargs)
 
-        # For the purpose of these cluster tools, it shouldn't make a significant difference
-        # whether we use spawn or fork. However, TensorFlow is not fork-safe, see:
-        # https://github.com/tensorflow/tensorflow/issues/5448#issuecomment-258934405
-        multiprocessing.set_start_method("spawn", force=True)
+        self.did_overwrite_start_method = False
+        if "start_method" in kwargs is not None:
```

Comment:
```suggestion
        if "start_method" in kwargs and kwargs["start_method"] is not None:
```

Reply:
Maybe you can add a test also for the start_method=None case, as this is how we'd use it in voxelytics.

Reply:
> Maybe you can add a test also for the start_method=None case, as this is how we'd use it in voxelytics.

I added that test now. However, it passed before this fix, since set_start_method was fine with being passed None. However, it's obviously cleaner to completely ignore the parameter if its value is None. Thank you for spotting this!